src/components/ui/RouteHeader.tsx

```diff
@@ -9,18 +9,19 @@
 import { cn } from "@stanfordspezi/spezi-web-design-system";
 import { createLink, type LinkComponentProps } from "@tanstack/react-router";
 import { ArrowLeft } from "lucide-react";
-import { type ComponentProps, type ReactNode } from "react";
+import { type ComponentProps, type ReactNode, type RefAttributes } from "react";
 import { useIsScrolled } from "@/utils/useIsScrolled";
 import { DashedSeparator } from "./DashedSeparator";
 
-interface RouteHeaderProps {
+interface RouteHeaderProps extends RefAttributes<HTMLDivElement> {
```

Comment:
I suggest this, as it makes it easier to get other properties if necessary. Fine as is though.  

```suggestion
interface RouteHeaderProps extends Pick<ComponentProps<"div">, "ref"> {
```